ctpolicy/ctpolicy.go

```diff
@@ -141,3 +157,24 @@ func (ctp *CTPolicy) GetSCTs(ctx context.Context, cert core.CertDER) (core.SCTDE
 	}
 	return ret, nil
 }
+
+// SubmitFinalCert submits finalized certificates created from precertificates
+// to any configured logs
+func (ctp *CTPolicy) SubmitFinalCert(cert []byte) {
+	isPrecert := false
```

Comment:
I think it would be clearer to do:

`falseVar := false`

Then, below, you'd be saying `Precert: &falseVar` rather than `Precert: &isPrecert`, which makes it more immediately obvious what the value is.